CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/ViewModels/BaseLineViewModel.cs

```diff
@@ -1,75 +1,49 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Collections.ObjectModel;
 using System.ComponentModel;
+using System.Linq;
 using CheckerUI.Helpers;
+using CheckerUI.Models;
 using Xamarin.Forms;
 
 namespace CheckerUI.ViewModels
 {
     public class BaseLineViewModel : BaseViewModel
     {
-        private static int m_counterID = 0; // static counter well be changed as db will contains pk id
+        private static int m_counterID = 0;
+        private static int m_LockedCounterID = 100;// static counter well be changed as db will contains pk id
 
         private Button buttonToMake;
         public Command FeelOrdersCommand;
-        public Dictionary<int, OrderItemView> m_Orders { get;set;} = new Dictionary<int, OrderItemView>();
+        public Dictionary<int, OrderItemView> m_Orders { get; set; } = new Dictionary<int, OrderItemView>();
         private Grid m_currentViewableGrid = null;
-        private readonly Grid m_GridToMake;
-        private readonly Grid m_GridToLoc;
-        private readonly Grid m_GridOrders;
-        private readonly Grid m_InProgressGrid;
-       
-        
-        private  List<OrderItemView> m_DoneOrdersList { get; set; } = new List<OrderItemView>();
+        private Grid m_GridOrders;
+
+        // public ObservableCollection<Button> m_ButtonsToMake = new ObservableCollection<Button>();
+        public ObservableCollection<OrderButtonModel> m_ButtonsInProgress = new ObservableCollection<OrderButtonModel>();
```

Comment:
same as above: when creating a property - something public - the naming convention is sans the prefix. imo